api/DOMRect.json

```diff
@@ -225,6 +225,104 @@
           }
         }
       },
+      "height": {
+        "__compat": {
+          "spec_url": "https://drafts.fxtf.org/geometry/#dom-domrectreadonly-height",
```

Comment:
@vinyldarkscratch this still has "readonly" in the URL. This file shouldn't link to those :)

Reply:
I will make the suggestions.

Reply:
```suggestion
          "spec_url": "https://drafts.fxtf.org/geometry/#dom-domrect-height",
```